frontend/src/components/pages/admin/Admin.DebugBundle.tsx

```diff
@@ -82,9 +82,14 @@ export class AdminDebugBundle extends Component<{}> {
         return (
             <Box>
                 <Header />
-                {api.canDownloadDebugBundle && <Text mt={4} fontWeight="bold">Latest debug bundle:</Text>}
-                <DebugBundleLink statuses={api.debugBundleStatuses} showDeleteButton />
-                {api.debugBundleStatuses.length === 0 && <Text>No debug bundle available for download.</Text>}
+                <Box mt={4}>
+                    {(api.canDownloadDebugBundle || api.isDebugBundleExpired) && <Text fontWeight="bold">Latest debug bundle:</Text>}
+                    {api.isDebugBundleExpired && <Text>Your previous bundle has expired and cannot be downloaded.</Text>}
+                    {api.isDebugBundleError && <Text>This debug bundle was cancelled by the user and is not available for download.</Text>}
+                    {api.canDownloadDebugBundle && <DebugBundleLink statuses={api.debugBundleStatuses} showDeleteButton/>}
```

Comment:
I am not sure if I understand React code completely ... Will they be able to delete an expired bundle with this logic? They should be able to.

Reply:
They can generate a new bundle in that case. If we want to keep an explicit delete action as well, I can add that. Just let me know cc @ivpanda.

Reply:
@bojand I'm assuming that when they generate a new one, the old one is overwritten? In this case, I'd prefer to not add a delete button— if a user sees that they can't download something, it's contradictory UX to tell them that they should delete it, too. I'd rather delete it for them on expiry if they can't download it.

Reply:
I've already added it before I saw this comment, but it's super easy to remove.